### PR TITLE
create a history in Chinese

### PR DESCRIPTION
**Pull request by Anoor2css**
Please review this, @casey-collab

### CODE REVIEW

File: Chinese/Donggewudi/Donggeheiheihei.md [hunk 1]
```diff
@@ -0,0 +1 @@
+孙悟空变成了东哥，就此解放了压抑数百年的欲望，看见石头就想嘿嘿嘿
```
Comment (casey-collab) [inline Chinese/Donggewudi/Donggeheiheihei.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Anoor2css** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**